### DISCUSSION TITLE
Accordion item content: add width and height css variables

### DISCUSSION THREAD

**Discussion by bogdan0083** [chakra-ui/zag#1258] [Ideas]
# 🚀 Feature request

It would be cool to be able to get item content `width` and `height` through css variables. This would allow us to create slide animations similar to new [Collapsible](https://zagjs.com/components/vue-sfc/collapsible) component.

## 🧱 Problem Statement / Justification

Right now it's quite a challenge to create slide animations on item content. We need to use `Transition` component and compute size by hand. Having css variables on each item content would make life easier.

## ✅ Proposed solution or API

The solution would be to use the approach similar to [Collapsible](https://zagjs.com/components/vue-sfc/collapsible) component. Compute size of each content on the initial mount and recompute it on open/close.

**Comment by segunadebayo** [comment]
The accordion and collapsible can be composed together in the userland code as shown here:
https://github.com/chakra-ui/zag/commit/41d8ba7d6baac6aea39a9adc70d79e83398a3e63

We'll update the docs to include patterns like this.
